.github/workflows/ci.yml

```diff
@@ -30,11 +30,14 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: 18
-          cache: 'pnpm'
+          cache: "pnpm"
 
       - name: Install dependencies
         run: pnpm install
 
+      - name: Format Check
```

Comment:
@dmitriiIdentityLabs sorry, I needed to add some additional changes like the step name here.